openstack/manila/alerts/openstack-manila.alerts

```diff
@@ -71,6 +71,33 @@ groups:
       description: Manila Shares taking more than 2 minutes to resize in {{ $labels.manila_host }}
       summary: Manila Shares taking more than 2 minutes to resize
 
+  - alert: OpenstackManilaShareServersStuckCreate
+    expr: max(openstack_manila_share_servers_stuck_max_duration_gauge{status="creating"})
+      BY (manila_host) > 120
+    for: 5m
+    labels:
+      dashboard: manila
+      service: manila
+      severity: info
+      tier: os
+    annotations:
+      description: Manila Share server {{ $labels.id }} taking more than 2 minutes to create on {{ $labels.manila_host }}
```

Comment:
Because of the grouping  (by manila_host) I think the `$labels.id` is not available here